packages/inference/src/providers/wavespeed.ts

```diff
@@ -188,13 +202,11 @@ export class WavespeedAIImageToImageTask extends WavespeedAITask implements Imag
 	}
 
 	async preparePayloadAsync(args: ImageToImageArgs): Promise<RequestArgs> {
-		return {
-			...args,
-			inputs: args.parameters?.prompt,
-			image: base64FromBytes(
-				new Uint8Array(args.inputs instanceof ArrayBuffer ? args.inputs : await (args.inputs as Blob).arrayBuffer())
-			),
-		};
+		const hasImages =
+			(args as Record<string, unknown>)["images"] ??
+			(args.parameters as Record<string, unknown> | undefined)?.["images"];
```

Comment:
better to use dot accessing here 
```suggestion
		const hasImages =
			(args as { images?: unknown }).images ?? (args.parameters as Record<string, unknown> | undefined)?.images;
```